includes/api/class-wc-admin-rest-reports-products-stats-controller.php

```diff
@@ -31,6 +31,13 @@ class WC_Admin_REST_Reports_Products_Stats_Controller extends WC_REST_Reports_Co
 	 */
 	protected $rest_base = 'reports/products/stats';
 
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		add_filter( 'woocommerce_reports_products_stats_select_query', array( $this, 'set_default_report_data' ), 10, 2 );
```

Comment:
If you're not using `$args` now, you can remove the `10, 2` here & fall back to the default value of 1 accepted arg (and default priority 10).

Reply:
Thanks, Kelly!  Removed.